tools/deprecated/generate-activity-metrics.py

```diff
@@ -52,7 +53,7 @@ def get_data(url, username, pw):
 
     return extract_json_response(res)
 
-def noon_of(day=datetime.now()):
+def noon_of(day=timezone.now()):
     return datetime(year=day.year, month=day.month, day=day.day, hour=12)
```

Comment:
hm, I think the intent here probably was in fact local time, so let's leave it as datetime.now